numpy/_core/numerictypes.py

```diff
@@ -95,9 +95,6 @@
 
 # we don't need all these imports, but we need to keep them for compatibility
 # for users using np._core.numerictypes.UPPER_TABLE
```

Comment:
The comment specifies why these need to remain.

Reply:
I seem to recall the F401 removals were automatic, then manually reviewed. Some fell through the cracks, I'll fix that. Is it OK to add `# noqa: F401` to these lines?